src/common/slices/snackbarSlice.jsx

```diff
@@ -0,0 +1,23 @@
+import {createSlice} from '@reduxjs/toolkit';
+
+const initialState = {
+  snackbarConfig: {
+    show: false,
+  },
```

Comment:
```suggestion
    show: false
```
No need to nest under `snackbarConfig` as slice is for snack bar only. snackbar.snackbarConfig.show is bit too long

Reply:
The slice is for `snackbarConfig`. The slice can be renamed.